gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri Sep 16 18:38:07 EDT 2016
+#Sat Sep 15 19:45:22 PDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
Ah, good this is a def a req for increasing your API level support. Gradle needs to be updated as you did here!

Reply:
Terrific! Welcome!

I have another cut at updating for Robolectric compatibility. I'll push that shortly after trying further to work through the errors that arise at each attempt.

Their [migration guide](http://robolectric.org/migrating/) is somewhat helpful. The good news is they've updated Robolectric for current or almost-current Android APIs. The bad news is various parts of their APIs changed and various other tools and libs have to upgrade to get there.

In general, the Android gradle rules (esp. for this project) and library dependencies are a mystery to me. Docs say we need to turn on `includeAndroidResources = true` but does that mean we should turn off the `build.workaround-missing-resource.gradle`?